internal/config/identity/openid/jwt.go

```diff
@@ -38,7 +37,7 @@ type publicKeys struct {
 	*sync.RWMutex
 
 	// map of kid to public key
-	pkMap map[string]crypto.PublicKey
+	pkMap map[string]interface{}
```

Comment:
Revert? It seems to compile fine without this change?

Reply:
no this change is needed otherwise it won't work to support multiple types of KEY inputs

- []byte
- crypto.PublicKey

For different types of Algos.

Reply:
`crypto.PublicKey` is `any`, so `[]byte` already converts to it.

Reply:
yeah but its not @klauspost since its typed.

Reply:
Also it confuses our library that does JWT verification it relies on the fact that `key` is of type that is internal

```
// Verify implements token verification for the SigningMethod. Returns nil if the signature is valid.
func (m *SigningMethodHMAC) Verify(signingString, signature string, key interface{}) error {
        // Verify the key is the right type
        keyBytes, ok := key.([]byte)
        if !ok {
                return ErrInvalidKeyType
        }
```